modules/apps/export-import/export-import-rest-api/src/main/java/com/liferay/exportimport/rest/resource/v1_0/ReportEntryResource.java

```diff
@@ -45,7 +46,7 @@
 @ProviderType
 public interface ReportEntryResource {
 
-	public Page<ReportEntry> getImportProcessErrorsPage(
+	public Page<ReportEntry> getImportProcessReportEntriesPage(
```

Comment:
The commit message is not correct:

` getImportProcessReportEntriesPage as not the endpoint is called` instead of 
` getImportProcessReportEntriesPage as now the endpoint is called`